Shared/Services/DeviceService.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  DeviceService.swift
+//  SyncExtension
+//
+//  Created by Richard Ascanio on 1/18/24.
+//
+
+import Foundation
+import InternxtSwiftCore
+
+struct DeviceService {
+    static var shared = DeviceService()
+    private let backupAPI: BackupAPI = APIFactory.Backup
+    private let decrypt = Decrypt()
+    private let config = ConfigLoader().get()
+
+    public func getAllDevices(deviceName: String?) async throws -> Array<Device> {
```

Comment:
You can create your own type here, and provide a boolean indicating which device is the current, this way the view that is going to display this, doesn't know the logic behind which device is the current one